src/components/profile/index.js

```diff
@@ -2,14 +2,39 @@
 
 /** @jsx jsx */
 import { jsx } from '@emotion/core'
-import React from 'react'
+import * as React from 'react'
+import type { BrowserHistory, HashHistory, MemoryHistory } from 'history'
+
 import ROUTE from 'constants/route'
 import InputField from 'components/common/input-field'
 
 import style from './index.style.js'
 
-export default class SignUp extends React.Component {
-  state = {}
+export type RouterHistory = BrowserHistory | HashHistory | MemoryHistory
+
+export type ProfilePropsType = {
```

Comment:
Usually, there is no need to export PropsType

Reply:
:boom: